index.html

```diff
@@ -0,0 +1,206 @@
+                        </div>
+                        <div class="doDivHead">
+                            <h5>plug Brand</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug brand is a branch of plug 
+                                store that we offer to our customers i.e office 
+                                branding, product stickers, postas.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="doDiv">
+                        <div class="doDivImg">
+                            <img src="img/camera.png" alt="Online Commerce">
+                        </div>
+                        <div class="doDivHead">
+                            <h5>Plug shots</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug shots is service offered to anyone
+                                who needs a photoshoot, or product photoshoot including 
+                                events.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="doDiv">
+                        <div class="doDivImg">
+                            <img src="img/video.png" alt="Online Commerce">
+                        </div>
+                        <div class="doDivHead">
+                            <h5>Plug vid</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug vid is a videography service offered to 
+                                our clients with events or product commercials to
+                                aired on tv
+                            </p>
+                        </div>
+                    </div>
+                    <div class="doDiv">
+                        <div class="doDivImg">
+                            <img src="img/software.png" alt="Online Commerce">
+                        </div>
+                        <div class="doDivHead">
+                            <h5>Plug soft</h5>
+                        </div>
+                        <div>
+                            <p class="doivpara">Plug soft is a tech service offered
+                                to clients who need websites, application for their
+                                bussiness.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="doInno">
+                    <p class="doInnoPara">Lets innovate and create</p>
+                </div>
+                <div class="doLink">
+                    <p> <a href="" class="doLink">Get more information about us</a></p>
+                </div>
+            </div>
+            <div class="teamBack" id="tea">
+
+            </div>
```

Comment:
**Changes Suggested**
- [ ] Let us remove the HTML5 `main` element and replace it with `section` element.
- [ ] Let us make the immediate child of the `main` element an `article` element.
- [ ] The background of this section doesn't resemble the given design. Let us try and get to the design as close as possible.
- [ ] Let us add padding to the paragraph sentences on the cards.
- [ ] The font size and the colors don't resemble those on the design. Let us try as much as possible to get to the design.